temporal-testing/src/main/resources/META-INF/native-image/io.temporal/temporal-testing/jni-config.json

```diff
@@ -0,0 +1,29 @@
+[
+{
+  "name":"java.lang.ClassLoader",
```

Comment:
It this file generated automatically using one of the Graal agents?

If so, it would be nice to get a Readme.md containing steps to generate/regenerate it.